Packages/vcs/Lib/utils.py

```diff
@@ -571,15 +572,15 @@ def scriptrun(script):
             except Exception,err:
               print "could not load tmpl:",nm,err
           else:
-            try:
+            #try:
               loadVCSItem(loader[typ],nm,v)
-            except Exception,err:
-              print "failed",typ,nm,err
-    except Exception,err: #ok could not read json file maybe it is an old initial.attributes
-      if os.path.split(script)[-1] == "initial.attributes":
-        _scriptrun(script)
-      else:
-        warnings.warn("unable to source file: %si %s" % (script,err))
+            #except Exception,err:
+            #  print "failed",typ,nm,err
+    #except Exception,err: #ok could not read json file maybe it is an old initial.attributes
```

Comment:
thanks. just wanted to make sure. Like I said if code is not useful remove it. Commenting a big chunk of code makes code unreadable and non-production like.